recipes-core/acrn/acrn-hypervisor.bb

```diff
@@ -8,6 +8,7 @@ ACRN_SCENARIO  ?= "sdc"
 
 EXTRA_OEMAKE += "HV_OBJDIR=${B}/hypervisor EFI_OBJDIR=${B}/efi-stub"
 EXTRA_OEMAKE += "BOARD=${ACRN_BOARD} FIRMWARE=${ACRN_FIRMWARE} SCENARIO=${ACRN_SCENARIO}"
+EXTRA_OEMAKE += "BOARD_FILE=${S}/misc/acrn-config/xmls/board-xmls/${ACRN_BOARD}.xml SCENARIO_FILE=${S}/misc/acrn-config/xmls/config-xmls/${ACRN_BOARD}/${ACRN_SCENARIO}.xml"
```

Comment:
would it be nice to have a saperate variable ACRN_SCENARIO_FILE for SCENARIO_FILE?   it would be case to set the SCENARIO_FILE to other xml file like industry_launch_2uos.xml or any customized xml file 
"${S}/misc/acrn-config/xmls/config-xmls/${ACRN_BOARD}/${ACRN_SCENARIO}.xml" good for default value